recipe/meta.yaml

```diff
@@ -1,24 +1,25 @@
 {% set org = "pyne" %}
-{% set version = "0.5.11" %}
+{% set version = "0.7.0" %}
 {% set build = "0" %}
-{% set sha256 = "ba05917b0de446ec520857655e77967a783e3bc723299faf5989f532d6b0647a" %}
-{% set variant = "openblas" %}
+{% set sha256 = "19e104a05aa56ef2e747f5e4a1c4cffefdb57a592a300823265ba72a1b55d4f3" %}
+#{% set variant = "openblas" %}
 
 package:
   name: pyne
   version: {{ version }}
 
 source:
   fn: pyne-{{ version }}-{{ sha256 }}.tar.gz
-  url: https://github.com/{{ org }}/pyne/archive/{{ version }}.tar.gz
+  url: https://github.com/pyne/pyne/archive/0.7.0-rc1.tar.gz
```

Comment:
You can keep the org and version variables in here for consistency, but would just have to add the '-rc1' at the end